### PR TITLE
Update scriptSelect.js

### LINKED ISSUE #54: Select field not updated when editing existing pages
**Initial Situation**
* Create a new page with a form that has a _Value_ and dependent _Select_ field
* Set the _Value_ field
* Set the _Select_ field
* Save the page
* Edit with form again

**Expected Result**
* Clicking on the _Select_ field provides the user with a list of options dependent on the already defined _Value_ field

**Current Result**
* The _Select_ field has only the one fixed value set before (not populated with any additional option)
* Note: Re-selecting the _Value_ field's value manually will trigger the refresh the _Select_ field

If we have a sandbox somewhere, I'd be more than happy to create a test case.

### PR DESCRIPTION
**Pull request by gesinn-it**
This closes #54.

### CODE REVIEW

**Comment by toniher** [comment]
@kghbln @mwjames sorry for the ignorance. If this would be merged, could it be tested straight at: https://sandbox.semantic-mediawiki.org/ ?
Thanks!

**Comment by mwjames** [comment]
>  If this would be merged, could it be tested straight at: 

Merged and in case the sandbox uses `@dev` then a 2.1.0-alpha should appear in the version list.

**Comment by kghbln** [comment]
> Merged and in case the sandbox uses @dev then 

[It does](https://sandbox.semantic-mediawiki.org/wiki/SMW_Sandbox:Setup_information) so happy testing.

File: res/scriptSelect.js [hunk 1]
```diff
@@ -299,16 +301,26 @@ function SFSelect_arrayEqual(a, b)
...
 	for (var i=0; i<SFSelect_fobjs.length; i++){
 
 		var fobj = SFSelect_fobjs[i];
-		var valuepat = "input[name='" + fobj.valuetemplate + "\\["+ fobj.valuefield + "\\]']";
+		
+		//var valuepat = "input[name='" + fobj.valuetemplate + "\\["+ fobj.valuefield + "\\]']";
+		
+		// hack to support multi instance templates: select all "select" items starting with fobj.valuetemplate
+		// example name attribute: name="myTemplate[0a][myField]"
+		valuepat = 'input[name^="' + fobj.valuetemplate + '"]';
```
Comment (mwjames) [inline res/scriptSelect.js]: Introduced a regression, fixed with https://github.com/SemanticMediaWiki/SemanticFormsSelect/commit/cdd36649d0b1d3ce4c3acf26732e0d5aa69c60e2.

Comment (toniher) [inline res/scriptSelect.js]: Thanks!

**Pull request merged**